Gordon360/Models/CCT/Match.cs

```diff
@@ -39,5 +35,8 @@ public partial class Match
         [ForeignKey("SurfaceID")]
         [InverseProperty("Match")]
         public virtual Surface Surface { get; set; }
+        [InverseProperty("Match")]
+        public virtual ICollection<MatchTeam> MatchTeam { get; set; }
```

Comment:
Generally, you might want to avoid naming properties of one class the same thing as another class name.

Reply:
Yes, but is the name of the property from a table column name that you created? That was what I was really suggesting you change, but I'm not exactly sure what is going on here.

Reply:
No, this is not tied to a column in the database. You can see the type of this property is an `ICollection<MatchTeam>`. It's a "Navigation Property" used to traverse the table relationships in C# without writing LINQ joins.